src/applications/search/styles.scss

```diff
@@ -195,7 +195,7 @@
       font-weight: 400;
     }
 
-    button {
+     button {
```

Comment:
Was this a weird linty thing? 🤔

Reply:
Changed it back, not sure what happened here